app/views/dradis/plugins/calculators/cvss/_addons_menu_bs4.html.erb

```diff
@@ -0,0 +1 @@
+<%= link_to 'Risk Calculators - CVSSv3', cvss_calculator.calculators_cvss_path, class: 'dropdown-item' %>
```

Comment:
@MattBudz isn't this missing the `<li>`? Also new line at the end. Merge afterwards